scripts/utils.js

```diff
@@ -6,14 +6,16 @@ const zlib = require('zlib');
 const jsesc = require('jsesc');
 const mkdirp = require('mkdirp');
 const regenerate = require('regenerate');
+const decodeRanges = require('../static/decode-ranges');
```

Comment:
```suggestion
const decodeRanges = require('../static/decode-ranges.js');
```

Reply:
I assume this also applies to the generated files, for example `symbols.js`:
```js
module.exports=require('./ranges'.js).flatMap(r=>Array.from(r.values()))
```
Is that correct?

Reply:
For the generated code, I care a little less, but yes let's be consistent